package.json

```diff
@@ -60,6 +60,7 @@
     "react-spinkit": "^1.1.11",
     "react-translate-component": "^0.11.1",
     "redux": "^3.6.0",
+    "redux-thunk": "^2.1.0",
```

Comment:
ok, I fixed the problem (and redux-thunk is now present in the shrinkwrap dump).
Note to the future: we should stop using the official NPM implementation to build shrinkwraps (as it's quite crappy - just see the `fsevents` stuff). When using the `npm-shrinkwrap` package (which you installed btw) everything works great, so we should run this package over our dependencies before commiting any changes to the `package.json` in the future